spec/factories.rb

```diff
@@ -1,2 +1,6 @@
 FactoryBot.define do
+  factory :income do
+    sequence(:name) { |n| "name#{n}" }
+    sequence(:frequency) { |n| "frequency#{n}" }
```

Comment:
Is frequency a number? 
Since it doesn't need to be unique, a suggestion to use a value that makes sense what this attribute is about

Reply:
Thanks Elle, no frequency is a word - weekly, monthly, quarterly...may make sense to have an enum, will take a look